dashboard/config/scripts/time4cs-experiment-unit-2.script

```diff
@@ -1,5 +1,6 @@
 login_required true
 hideable_stages true
+pilot_experiment 'time4cs-experiment'
```

Comment:
I feel like I remember broward having 2 scripts. Does the other one need to be updated?

Reply:
Hey Dani, they actually have 4, but the other 3 were already marked with this pilot for some reason.

Reply:
Ah ok!